Week3/homework/CreditCardValidate.js

```diff
@@ -0,0 +1,46 @@
+"use strict"
+
+function validCreditCard (num) {
+    let card = num;
+    let cardNumber = card.split("");
+    let cardNumberDif = cardNumber.filter((este, i) => cardNumber.indexOf(este) === i);
+    let cardNumberLen = cardNumberDif.length;
+    let soma = 0;
+         /* My intention here is to define that the digits can't be all the same*/
+        if (cardNumberLen = 1) {
+            return `The card ${card} is invalid!`
+        }
+        /* Here I wanted to make sure that the sum of all digits are not less than 16 */
+        for (let i = 0; i < cardNumber.length; i++){
+            soma += cardNumber[i]; 
+                if (soma < 16){
+                return `The sum of the digits of the card ${card} is less than 16!`
+                }
+        }
```

Comment:
👍 for logic but I don't think `soma` is a correct variable name as it doesn't describe what value it contains.